internal/command/user_machine.go

```diff
@@ -21,6 +21,8 @@ type AddMachine struct {
 type Machine struct {
 	models.ObjectRoot
 
+	// ID is optional, if empty it will be generated
+	ID              string
```

Comment:
You don't have to create a new attribute as the ID is used as AggregateID anway, so you can fill in the AggregateID if provided by the request.